pages/[id]/bridge.tsx

```diff
@@ -204,7 +211,7 @@ const Bridge = () => {
   // Define new variables that help control UI states and display pretty error strings
   const { errorType, errorReason } = formatError(error);
   // Helpers for different parts of UI state
-  const isAmountError = errorType === ErrorType.ERC20AmountError;
+  const isAmountError = errorType === ErrorType.ERC20AmountError || !isSufficientBalance;
```

Comment:
in the case of `!isSufficientBalance`, the `errorType` is already `ErrorType.ERC20AmountError` so we can make this shorter

```suggestion
  const isAmountError = errorType === ErrorType.ERC20AmountError;
```

Reply:
https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/129/commits/1f438166e417b47324f54d81c9704c19638407b4